index.d.ts

```diff
@@ -1,3 +1,7 @@
+declare const stop: unique symbol;
+
+export type StopSymbol = typeof stop;
```

Comment:
I don't see `StopSymbol` used anywhere.

Reply:
Based on your idea above of not exposing the existence of `{ [stop] }` in the returned object, perhaps this is indeed not needed. The reason I first added this was to have a way to expose it to the user, since [for example in find-up it is also exposed](https://github.com/sindresorhus/find-up/blob/64d2422cd2d256c72049851ae4fa02413c814b7f/index.d.ts#L9).

Reply:
Should I undo this, then?

Reply:
> Should I undo this, then?

Yes

Reply:
What do you think about passing options param for exposing stop in the returned object?